KavitaEmail/KavitaEmail.csproj

```diff
@@ -6,6 +6,7 @@
         <RootNamespace>Skeleton</RootNamespace>
         <AssemblyVersion>0.1.13.0</AssemblyVersion>
         <LangVersion>11</LangVersion>
+        <UserSecretsId>6c7fdbb8-0db9-4c2c-843e-fa70bdd25264</UserSecretsId>
```

Comment:
What is this?

Reply:
I honestly have no clue. I didn't manually add it. Must be something added from build? Let me remove and see if it still works.

Reply:
Works without it. Resolving comment.